shared/src/main/scala/coop/rchain/store/LmdbKeyValueStore.scala

```diff
@@ -58,7 +58,13 @@ final case class LmdbKeyValueStore[F[_]: Sync](
   // GET
   override def get[T](keys: Seq[ByteBuffer], fromBuffer: ByteBuffer => T): F[Seq[Option[T]]] =
     withTxnSingleThread(isWrite = false) { (txn, dbi) =>
-      keys.map(x => Option(dbi.get(txn, x)).map(fromBuffer))
+      keys.map(
+        x =>
+          Try(dbi.get(txn, x))
+            .collect { case null => throw new Exception("Not found"); case b: ByteBuffer => b }
+            .toOption
+            .map(fromBuffer)
```

Comment:
`Option(null) == None` is wrapping null value to Option.

With `Try(...).toOption` we are basically swallowing any exception to None.

Please refer to this issue how to handle exceptions.
- #3769

Reply:
> `Option(null) == None` is wrapping null value to Option.
> 
> With `Try(...).toOption` we are basically swallowing any exception to None.
> 
> Please refer to this issue how to handle exceptions.
> 
>     * [Strategy, categorization and handling exceptions #3769](https://github.com/rchain/rchain/issues/3769)

We shouldn't swallow exceptions, I agree. But in that case we should change interface of `KeyValueStore.get` to return sequence of `Either` or `Try` and all of three derivations.

I think returning `None` from `get` when something goes wrong is expected behavior.

Reply:
We don't want to handle errors in any way and leave errors to _bubble up_ in higher layers where we want to decide what is appropriate action.

Reply:
Higher layer is `WebApiRoutes.scala` in that case. It is Ok to print exception with stack trace as is for admin. But how to convert this exception to something meaningful for client?